src/app/data-planes/index.ts

```diff
@@ -4,14 +4,10 @@ import { token } from '@/services/utils'
 export * from './routes'
 
 type Token = ReturnType<typeof token>
-type Sources = ReturnType<typeof sources>
 
-const $ = {
-  sources: token<Sources>('mesh.sources'),
-}
 export const services = (app: Record<string, Token>): ServiceDefinition[] => {
   return [
-    [$.sources, {
+    [token('data-planes.sources'), {
```

Comment:
Again, all of these (and further down) are because:

1. I don't want to export these tokens
2. Its almost pointless typing them as they should use the type of the label (they don't, yet)